frontend/src/utilities/useWatchConsoleLinks.tsx

```diff
@@ -21,7 +21,7 @@ export const useWatchConsoleLinks = (): ConsoleLinkResults => {
   });
 
   React.useEffect(() => {
-    let watchHandle: ReturnType<typeof setTimeout>;
+    let watchHandle: ReturnType<typeof setTimeout> | undefined;
```

Comment:
Remove the if condition in the return statement.
```suggestion
    let watchHandle: ReturnType<typeof setTimeout>;
```